payments-core/src/main/java/com/stripe/android/googlepaylauncher/injection/GooglePayPaymentMethodLauncherFactory.kt

```diff
@@ -1,15 +1,20 @@
-package com.stripe.android.googlepaylauncher
+package com.stripe.android.googlepaylauncher.injection
 
 import androidx.activity.result.ActivityResultLauncher
+import androidx.annotation.RestrictTo
+import com.stripe.android.googlepaylauncher.GooglePayPaymentMethodLauncher
+import com.stripe.android.googlepaylauncher.GooglePayPaymentMethodLauncherContract
 import dagger.assisted.AssistedFactory
 import kotlinx.coroutines.CoroutineScope
 
 @AssistedFactory
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
IUUC this will be changed back to internal right? if so please add a TODO here, ditto to `GooglePayLauncherModule`

Reply:
No, we'll keep using this from Payment Sheet. The change we discussed is included here, which was using this instead launching with `GooglePayPaymentMethodLauncherContract` directly.